packages/ethers5/src/client.ts

```diff
@@ -35,7 +35,7 @@ import type { EthereumProviderOptions } from '@walletconnect/ethereum-provider'
 // -- Types ---------------------------------------------------------------------
 export interface Web3ModalClientOptions extends Omit<LibraryOptions, 'defaultChain' | 'tokens'> {
   ethersConfig: ProviderType
-  siweConfig?: Web3ModalSIWEClient
+  siweConfig?: SIWEControllerClient
```

Comment:
Curious why you had to make that change ? `siweConfig` should be of type `Web3ModalSIWEClient`

Reply:
@Cali93 Seems like both types do not match:

```bash
@web3modal/ethers5:build: src/client.ts(251,7): error TS2322: Type 'Web3ModalSIWEClient | undefined' is not assignable to type 'SIWEControllerClient | undefined'.
@web3modal/ethers5:build:   Property 'signIn' is missing in type 'Web3ModalSIWEClient' but required in type 'SIWEControllerClient'.
```

```ts
export declare class Web3ModalSIWEClient {
    options: SIWEControllerClient['options'];
    methods: SIWEClientMethods;
    constructor(siweConfig: SIWEConfig);
    getNonce(): Promise<string>;
    createMessage(args: SIWECreateMessageArgs): string;
    verifyMessage(args: SIWEVerifyMessageArgs): Promise<true>;
    getSession(): Promise<import("@web3modal/core").SIWESession>;
    signOut(): Promise<boolean>;
}

export interface SIWEControllerClient extends SIWEClientMethods {
    signIn: () => Promise<SIWESession>;
    options: {
        enabled: boolean;
        nonceRefetchIntervalMs: number;
        sessionRefetchIntervalMs: number;
        signOutOnDisconnect: boolean;
        signOutOnAccountChange: boolean;
        signOutOnNetworkChange: boolean;
    };
}
```

I guess you have more context, can you check this when have a time?

Reply:
Reverted changes back since shouldn't be this PRs responsibility. Also the CI is not failing with builds so might be related to my environment